srcpkgs/encrypted-dns/INSTALL

```diff
@@ -0,0 +1,10 @@
+case "$ACTION" in
+    post)
+        if [ "$UPDATE" = "yes" ] ; then
+            :
+        else
+            chown _encrypted_dns:_encrypted_dns var/lib/encrypted-dns
+            chmod 750 var/lib/encrypted-dns
```

Comment:
use `make_dirs`, but this may already be the default for a system account's homedir

Reply:
chown is indeed not needed, removed.
chmod is still needed, because /var/lib/encrypted-dns should not be world-readable per package documentation.
make_dirs does not work here, because _encrypted_dns_homedir still installes as 755 and make_dirs will not override this.